pmd-java/src/main/java/net/sourceforge/pmd/lang/java/rule/codestyle/LinguisticNamingRule.java

```diff
@@ -35,20 +37,25 @@
             .defaultValue(false)
             .desc("Check return type of methods which contain the configured infix in their name (see transformMethodNames property).")
             .uiOrder(4.0f).build();
-    private static final StringMultiProperty BOOLEAN_METHOD_PREFIXES_PROPERTY = StringMultiProperty
-            .named("booleanMethodPrefixes").defaultValues("is", "has", "can", "have", "will", "should")
-            .desc("The prefixes of methods that return boolean.").uiOrder(5.0f).build();
-    private static final StringMultiProperty TRANSFORM_METHOD_NAMES_PROPERTY = StringMultiProperty
-            .named("transformMethodNames").defaultValues("to", "as")
-            .desc("The prefixes and infixes that indicate a transform method.").uiOrder(6.0f).build();
 
     private static final BooleanProperty CHECK_FIELDS = BooleanProperty.named("checkFields").defaultValue(true)
             .desc("Check field names and types for inconsistent naming.").uiOrder(7.0f).build();
     private static final BooleanProperty CHECK_VARIABLES = BooleanProperty.named("checkVariables").defaultValue(true)
             .desc("Check local variable names and types for inconsistent naming.").uiOrder(8.0f).build();
-    private static final StringMultiProperty BOOLEAN_FIELD_PREFIXES_PROPERTY = StringMultiProperty
-            .named("booleanFieldPrefixes").defaultValues("is", "has", "can", "have", "will", "should")
-            .desc("The prefixes of fields and variables that indicate boolean.").uiOrder(9.0f).build();
+
+    private static final PropertyDescriptor<List<String>> BOOLEAN_METHOD_PREFIXES_PROPERTY =
+            stringListProperty("booleanMethodPrefixes")
+                    .desc("The prefixes of methods that return boolean.")
+                    .defaultValues("is", "has", "can", "have", "will", "should").build();
```

Comment:
I think, here and for the other stringListProperties we need a `delim(",")` call.

Reply:
Since the delimiter was not set on the removed builders, it [defaults to "|"](https://github.com/pmd/pmd/blob/master/pmd-core/src/main/java/net/sourceforge/pmd/properties/MultiValuePropertyDescriptor.java#L28), which is also honored as the default in those new builders